src/modules/unpackfs/main.py

```diff
@@ -258,14 +285,14 @@ def run():
     root_mount_point = globalstorage.value("rootMountPoint")
 
     if not root_mount_point:
-        return ("No mount point for root partition in globalstorage",
+        return ("No mount point for root partition in globalstorage, "
                 "globalstorage does not contain a \"rootMountPoint\" key, "
                 "doing nothing")
 
     if not os.path.exists(root_mount_point):
-        return ("Bad mount point for root partition in globalstorage",
-                "globalstorage[\"rootMountPoint\"] is \"{}\", which does not "
-                "exist, doing nothing".format(root_mount_point))
+        return ("Bad mount point for root partition in globalstorage, "
+                "globalstorage[\"rootMountPoint\"] is \"{}\", which does "
```

Comment:
This is wrong, you're changing a 2-tuple return (errorstring, explanationstring) to a single string. Leave the comma where it was.